src/index.ts

```diff
@@ -27,6 +27,57 @@ export function start() {
             }
         })
     );
+
+    const informationMessageTestCommand = {
+        id: 'backend -plugin-information-message-command',
```

Comment:
it seems there is a extra space between backend and plugin all over the file ?

Reply:
A paste typo. Done.